.github/workflows/build-checks.yaml

```diff
@@ -99,6 +99,10 @@ jobs:
         run: |
           XDG_RUNTIME_DIR=$(mktemp -d "/tmp/kata-tests-$USER.XXX" | tee >(xargs chmod 0700))
           echo "XDG_RUNTIME_DIR=${XDG_RUNTIME_DIR}" >> "$GITHUB_ENV"
+      - name: Skip tests that depend on virtualization capable runners when needed
+        if: ${{ inputs.instance == 'ubuntu-22.04-arm' }}
```

Comment:
There's also a 24.04-arm instance - maybe make this future-proof with

```suggestion
        if: ${{ endsWith(inputs.instance, '-arm') }}
```

Reply:
Done, thanks for the suggestion!